src/FlowerShop.Core/Entities/Cart.cs

```diff
@@ -6,7 +6,7 @@ namespace FlowerShop.Core.Entities
     public class Cart
     {
         public Guid Id { get; private set; }
-        public decimal Price { get; private set; }
+        public decimal Price { get; set; }
```

Comment:
Entity setters should be left as private to encapsulate the logic. Also this allows us to follow Domain driven design.